ui/page/settings_page.go

```diff
@@ -433,6 +442,15 @@ func (pg *SettingsPage) Handle() {
 		pg.wal.SaveConfigValueForKey(dcrlibwallet.BeepNewBlocksConfigKey, pg.beepNewBlocks.IsChecked())
 	}
 
+	if pg.proposalNotification.Changed() {
+		pg.wal.SaveConfigValueForKey("proposalnotificationkey", pg.proposalNotification.IsChecked())
```

Comment:
`proposalnotificationkey` move this to load package utils.go file

```suggestion
		pg.wal.SaveConfigValueForKey(load.ProposalNotificationConfigkey, pg.proposalNotification.IsChecked())
```